iam-login-service/src/main/webapp/resources/iam/apps/dashboard-app/components/clients/client/client.component.html

```diff
@@ -1,90 +1,92 @@
-                        </uib-tab>
-
-                        <uib-tab index="4">
-                            <uib-tab-heading>Grant types</uib-tab-heading>
-                            <granttypelist client="$ctrl.clientVal" limited="false" />
-                        </uib-tab>
-
-                        <uib-tab index="5">
-                            <uib-tab-heading>Tokens</uib-tab-heading>
-                            <tokensettings client="$ctrl.clientVal"></tokensettings>
-                        </uib-tab>
-
-                        <uib-tab index="6">
-                            <uib-tab-heading>Crypto</uib-tab-heading>
-                            <clientcryptosettings client="$ctrl.clientVal"></clientcryptosettings>
-                        </uib-tab>
-
-                        <uib-tab index="7">
-                            <uib-tab-heading>Other info</uib-tab-heading>
-                            <othersettings client="$ctrl.clientVal"></othersettings>
-                        </uib-tab>
-
-                        <uib-tab index="8" ng-hide="$ctrl.newClient">
-                            <uib-tab-heading>Owners</uib-tab-heading>
-                            <client-owners client="$ctrl.clientVal" client-owners="$ctrl.clientOwners"></client-owners>
-                        </uib-tab>
-                    </uib-tabset>
-                </form>
-                <div class="form-group">
-                    <button class="btn btn-success" ng-click="$ctrl.saveClient()">Save client</button>
-                    <button class="btn btn-danger" ng-click="$ctrl.deleteClient()" ng-if="!$ctrl.newClient">Delete
-                        client</button>
-                    <button class="btn btn-default" ng-click="$ctrl.resetVal()" ng-if="!$ctrl.newClient">Reset</button>
-                    <button class="btn btn-default" ng-click="$ctrl.cancel()">Cancel</button>
-                </div>
-            </section>
-        </div>
-    </section>
-
-
+<!--
+
+    Copyright (c) Istituto Nazionale di Fisica Nucleare (INFN). 2016-2021
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<div>
+    <section class="content-header" ng-cloak>
+        <h1>
+            <i class="fa fa-rocket"></i>&nbsp;&nbsp; <span
+                ng-if="!$ctrl.newClient">{{$ctrl.clientVal.client_name}}</span>
+            <span ng-if="!$ctrl.newClient && !$ctrl.clientVal.active"> (Suspended)</span>    

```

Comment:
Change 1

Reply:
You can format this suspended text into a colored label

Reply:
![image](https://github.com/indigo-iam/iam/assets/99975605/0bd6739b-38a0-474b-a648-a500406ca380)

Reply:
![image](https://github.com/indigo-iam/iam/assets/99975605/507d980a-7b10-4805-b055-6689655ebf4f)

Reply:
![image](https://github.com/indigo-iam/iam/assets/99975605/88e669ee-3b77-4d76-b51a-4914d8e5d16e)

- I have changed the style of `Suspended` label. 

- On hover you can have the tooltip with more details of the suspension.